Pull Request 4

PatGarza20 opened the pull request:
Added new unit tests for LineProcessor to verify that its various line processing functions work correctly. Also added new GitRemote test to check its hashcode encoding function. Finally, added comments to the tests to make life easier for reviewers.

PatGarza20 committed:
Pull Request 4

Added new unit tests for LineProcessor to verify that its various line processing functions work correctly. Also added new GitRemote test to check its hashcode encoding function. Finally, added comments to the tests to make life easier for reviewers.

kyledewey commented:
Assigned @mj-shi

mj-shi reviewed (approved):
Tests are straightforward, simple and explained well for anyone to understand. Code is organized and spaced in a way that makes it easy to read.

PatGarza20 committed:
Pull Request 5

Added new unit tests for StringExtensions intended to verify whether its many string parsing methods function properly. Also added two more tests to LineProcessorTests so as to verify that empty Strings throw exceptions in certain methods.  A noticably large amount of code in this extension is heavy on dependencies so a lot of research will likely be necessary to test more complex classes.

PatGarza20 committed:
Pull Request 5 Mistake

I'm really tired of accidentally committing to a pull request that isn't merged yet
